eclipse-scout-core/src/tile/TileGridSelectionHandler.ts

```diff
@@ -205,41 +228,47 @@ export class TileGridSelectionHandler {
   }
 
   computeSelectionX(xDiff: number, extend: boolean): TileGridSelectionInstruction {
-    let tiles = this.getVisibleTiles();
-    let focusedTile = null;
-    let focusedTileIndex = -1;
-    let result = this._computeFocusedTile(xDiff);
+    let result = this._computeFocusedTileOrSelection(xDiff);
     if (result.selectedTiles !== null) {
```

Comment:
```suggestion
    if (result.selectedTiles) {
```

Or should we even need to check the length of the array? Maybe a JsDoc on the interface would help clarify the API.

Same with computeSelectionY.